horovod/common/operations.cc

```diff
@@ -865,6 +899,21 @@ void PerformOperation(TensorTable& tensor_table, MPIResponse response) {
 
         ACTIVITY_END_ALL(entries, timeline)
       }
+#elif HOROVOD_GPU_ALLREDUCE == 'D'
+      if (!horovod_global.ddl_initialized) {
+        // Initialize DDL
+        auto ddl_options = std::getenv("DDL_OPTIONS");
+        if (ddl_options == nullptr) {
+          throw std::logic_error("DDL_OPTIONS env variable needs "
```

Comment:
Similar to new the other comment, can this use mechanism similar to `DDL_CHECK`?